src/components/forms/form-group/FormGroupCtrl.ts

```diff
@@ -24,17 +24,45 @@ export class FormGroupCtrl {
 
   type: string;
   multiValue: boolean;
+  multiSelect: boolean;
+  multiSettings: any;
   option: string;
   parentId: Function = generateFieldId;
 
   constructor() {
+    this.multiSettings = {
+      template: "{{option}}",
+      smartButtonTextConverter(skip: any, option: any): any {
+        return option;
+      }
+    };
+    this.type = getTypeModelType(this.meta);
+    if (this.meta.type.TYPE_MODEL_VALUE === "LIST") {
```

Comment:
Hmm strange. I guess that when the JSON is parsed it is assuming that the list is a String, not an array, for some reason. What does the DMR output look like?